kakao/KakaoLoginIOS/KakaoLoginIOS/KakaoLoginIOS-Bridging-Header.h

```diff
@@ -0,0 +1,13 @@
+//
+//  KakaoLoginIOS-Bridging-Header.h
+//  KakaoLoginIOS
+//
+//  Created by Kevin Kang on 2017. 9. 15..
+//  Copyright © 2017년 Hara Kang. All rights reserved.
+//
+
+#ifndef KakaoLoginIOS_Bridging_Header_h
+#define KakaoLoginIOS_Bridging_Header_h
+
+#import <KakaoOpenSDK/KakaoOpenSDK.h>
```

Comment:
If modules are enabled in the project, you should be able to import the framework directly into Swift and won't need a bridging header.

For sample purposes though, this isn't a big concern.

Reply:
I'm sorry that I cannot understand your comment well. KakaoOpenSDK.framework source was written with Objective-C. Our project need this header file. Do you have another solution?